apps/myaccount/src/components/multi-factor-authentication/authenticators/fido-authenticator.tsx

```diff
@@ -79,9 +79,28 @@ export const FIDOAuthenticator: React.FunctionComponent<FIDOAuthenticatorProps>
     const [ editFIDO, setEditFido ] = useState<Map<string, boolean>>();
     const [ deleteKey, setDeleteKey ] = useState<string>("");
     const [ fidoFlowStartResponse, setFidoFlowStartResponse ] = useState<any>(null);
+    const [ oldFidoFlowStartResponse, setOldFidoFlowStartResponse ] = useState<any>(null);
 
-    const tryOlderDevice = useRef(false);
-    const retryAttempt = useRef(0);
+    // Safari doesn't allow the flow to be initiated in async.
+    // link https://bugs.webkit.org/show_bug.cgi?id=213595
+    // So the request is sent on component mount so that the button click can trigger the flow directly.
+    useEffect(() => {
+        if (new UAParser().getBrowser().name !== "Safari") {
+            return;
+        }
+
+        startFidoUsernamelessFlow()
+            .then((response) => {
+                setFidoFlowStartResponse(response);
+            });
```

Comment:
Don't we have to handle exceptions here?

Reply:
If we want to silent the errors shall we add a placeholder to add the debug logs like https://github.com/wso2/identity-apps/blob/4deb13dab9c4530d110e7074415dc25a1c970587/modules/react-components/src/components/containers/iframe.tsx#L172

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/2846/commits/89246b5a67347a495dd3f750e51d43197c29df06